extensions/amp-form/0.1/amp-form.js

```diff
@@ -64,6 +64,17 @@ export class AmpForm {
           'form action-xhr should not be on cdn.ampproject.org: %s',
           this.form_);
     }
+
+    const submitButtons = this.form_.querySelectorAll('input[type=submit]');
```

Comment:
I think it would be more robust and efficient to re-query these when you need them